public/app/features/dashboard/components/GenAI/GenAIDashboardChangesButton.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useMemo } from 'react';
+
+import { DashboardModel } from '../../state';
+import { Diffs, jsonDiff } from '../VersionHistory/utils';
+
+import { GenAIButton } from './GenAIButton';
+import { Message, Role } from './utils';
+
+interface GenAIDashboardChangesButtonProps {
+  dashboard: DashboardModel;
+  onGenerate: (title: string, isDone: boolean) => void;
+}
+
+const CHANGES_GENERATION_STANDARD_PROMPT = [
+  'You are an expert in Grafana Dashboards',
+  'Your goal is to write a description of the changes for a dashboard',
+  'When refering to panel changes, use the panel title',
+  'When using panel title, wrap it with double quotes',
+  'Group all the positioning changes together under the title "Panel position changes"',
+  'Group changes when all panels are affected',
+  'Do not mention line number',
+  'Refer to templating elements as variables',
+  'Ignore and never mention "getVariables" property changes',
+  'Ignore and never mention changes about plugin version',
+  'Try to make it as short as possible.',
+].join('. ');
+
+export const GenAIDashboardChangesButton = ({ dashboard, onGenerate }: GenAIDashboardChangesButtonProps) => {
+  const messages = useMemo(() => getMessages(dashboard), [dashboard]);
+
+  return <GenAIButton messages={messages} onReply={onGenerate} loadingText={'Generating title'} temperature={0} />;
```

Comment:
```suggestion
  return <GenAIButton messages={messages} onReply={onGenerate} loadingText={'Generating changes summary'} temperature={0} />;
```

or something along those lines?